packages/material-ui/src/styles/experimentalStyled.js

```diff
@@ -1,6 +1,6 @@
 import styled from '@material-ui/styled-engine';
 import { propsToClassKey } from '@material-ui/styles';
-import { styleFunctionSx } from '../Box/styleFunction';
+import { styleFunctionSx } from '@material-ui/system';
```

Comment:
I agree with Sebastian, better make it internal, for now.

```suggestion
import { unstable_styleFunctionSx } from '@material-ui/system';
```